### ISSUE TITLE
Bulk uploader not able to mass convert trees to empty planting sites 

### ISSUE THREAD

**Issue by hadleystein1** [OpenTreeMap/otm-core#2622]
Due to a bug in the iOS application, it's impossible to add an Empty Planting Site. Adding no information instead converts the record to a tree with missing species (https://github.com/OpenTreeMap/otm-ios/issues/304)

Attempted to mass update tree records with missing species and diameter information on the Arlington map. Ran an advanced search for trees with missing species and diameter information (see [here](https://www.opentreemap.org/arlington/map/?z=13/42.4115/-71.1510&q=%7B%22species.id%22%3A%7B%22ISNULL%22%3Atrue%7D%2C%22tree.diameter%22%3A%7B%22ISNULL%22%3Atrue%7D%7D), exported the list and changed the tree present field from TRUE to FALSE. 
[reupload_Arlington_MA.xlsx](https://github.com/OpenTreeMap/otm-core/files/312101/reupload_Arlington_MA.xlsx) 

The mass update appeared successful (https://www.opentreemap.org/arlington/management/bulk-uploader/), however, none of the trees with missing data converted to empty planting sites.

**Comment by jwalgran** [comment]
This is the expected, but perhaps not the desired, behavior of the importer. Passing a `False` value for "Tree Preset" does not delete existing tree data, it only prevents tree data from being created.

https://github.com/OpenTreeMap/otm-core/blob/9f49ddf638a0ce8ecac0ace2a02f5a0b641edd71/opentreemap/importer/models/trees.py#L170-L183

Question: Should the importer delete a tree if a row in the CSV has a "Planting Site ID" matching an existing map feature and "Tree Present" set to False?

**Comment by hadleystein1** [comment]
To answer your question, I think if a CSV row has a Planting Site ID matching an existing record on the map and the Tree Present field is set to False it should convert to an Empty Planting Site.

Question: Is there any way to mass convert trees that were improperly added due to a bug (but also trees in general) into empty planting sites using the bulk importer?

**Comment by RobinIsTheBird** [comment]
I wonder whether this bug either duplicates or depends upon https://github.com/OpenTreeMap/otm-core/issues/2638, Bulk uploader error message re: tree count when updating trees.

The root cause of that bug appears to be that the planting site id gets dropped somewhere along the way. So bulk update might be broken in general.

**Comment by RickMohr** [comment]
@RobinIsTheBird, I think that's a different issue. We prevent you from uploading more rows than you have available trees in your plan, but didn't consider that rows with a planting site id shouldn't be counted. Should be an easy fix to the counting code.

**Comment by jwalgran** [comment]
We determined that the iOS application was correctly creating empty planting sites, but want incorrectly displaying them with a "Missing Species" label. This was fixed in https://github.com/OpenTreeMap/otm-ios/pull/308

We already have a developer tool to mass delete all tree data on a map. I reviewed the issue with @dboyer and decided that it is somewhat dangerous to allow bulk deletion of trees via the importer, and the situations in which it would be required are rare. We are opting to not implement this.
